### PR TITLE
protoc-gen-grpc-web: Add version 1.0.7

### PR DESCRIPTION
**Pull request by hanabi1224**

### CODE REVIEW

File: bucket/protoc-gen-grpc-web.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.7",
+    "description": "gRPC for Web Clients",
+    "homepage": "https://github.com/grpc/grpc-web/",
+    "license": "Apache-2.0",
+    "url": "https://github.com/grpc/grpc-web/releases/download/1.0.7/protoc-gen-grpc-web-1.0.7-windows-x86_64.exe",
+    "hash": "ee9bac4b54ff1db787f3546355116730178aadb691e175740377943f8273ef50",
+    "pre_install": "Move-Item \"$dir\\protoc-gen-grpc-web-*.exe\" \"$dir\\protoc-gen-grpc-web.exe\"",
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
```

File: bucket/protoc-gen-grpc-web.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.7",
+    "description": "gRPC for Web Clients",
+    "homepage": "https://github.com/grpc/grpc-web/",
+    "license": "Apache-2.0",
+    "url": "https://github.com/grpc/grpc-web/releases/download/1.0.7/protoc-gen-grpc-web-1.0.7-windows-x86_64.exe",
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/grpc/grpc-web/releases/download/1.0.7/protoc-gen-grpc-web-1.0.7-windows-x86_64.exe#/protoc-gen-grpc-web.exe",
```

File: bucket/protoc-gen-grpc-web.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "hash": "ee9bac4b54ff1db787f3546355116730178aadb691e175740377943f8273ef50",
+    "pre_install": "Move-Item \"$dir\\protoc-gen-grpc-web-*.exe\" \"$dir\\protoc-gen-grpc-web.exe\"",
+    "bin": "protoc-gen-grpc-web.exe",
+    "checkver": {
+        "github": "https://github.com/grpc/grpc-web/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/grpc/grpc-web/releases/download/$version/protoc-gen-grpc-web-$version-windows-x86_64.exe",
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/grpc/grpc-web/releases/download/$version/protoc-gen-grpc-web-$version-windows-x86_64.exe#/protoc-gen-grpc-web.exe"
            }
        },

File: bucket/protoc-gen-grpc-web.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.7",
+    "description": "gRPC for Web Clients",
+    "homepage": "https://github.com/grpc/grpc-web/",
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
    "homepage": "https://grpc.io",
```

File: bucket/protoc-gen-grpc-web.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "Apache-2.0",
+    "url": "https://github.com/grpc/grpc-web/releases/download/1.0.7/protoc-gen-grpc-web-1.0.7-windows-x86_64.exe",
+    "hash": "ee9bac4b54ff1db787f3546355116730178aadb691e175740377943f8273ef50",
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
            "hash": "ee9bac4b54ff1db787f3546355116730178aadb691e175740377943f8273ef50"
        }
    },
```

File: bucket/protoc-gen-grpc-web.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    "pre_install": "Move-Item \"$dir\\protoc-gen-grpc-web-*.exe\" \"$dir\\protoc-gen-grpc-web.exe\"",
+    "bin": "protoc-gen-grpc-web.exe",
+    "checkver": {
+        "github": "https://github.com/grpc/grpc-web/"
```
Comment (Ash258) [inline bucket/protoc-gen-grpc-web.json]:
```suggestion
        "github": "https://github.com/grpc/grpc-web"
```

**Comment by hanabi1224** [comment]
@Ash258 Fixed in https://github.com/ScoopInstaller/Main/pull/839

**Pull request closed without merging**